### PR TITLE
Replace index page with Genten relevant content

### PR DESCRIPTION
**Pull request by apachano**
Just adding some content / getting all the git stuff started.

### CODE REVIEW

File: app/templates/index.html [hunk 1]
```diff
@@ -2,7 +2,17 @@
...
+    <p class="lead">Genten Studios is a group of programmers who are passionate about creating games. All of our games
+        follow a common pattern that the program itself does not provide any content. Content is defined by modules
+        written in Lua which the game does the heavy lifting turning it into something playable. In the long-term our
+        goal is to create Quartz Engine to be the core of the games we create in this style.</p>
+    <h2>Project Phoenix</h2>
+    <p>Our project coded "Phoenix" is the first game that we are developing in this style. Phoenix is an open world
+        sandbox game where the player can manipulate anything in a cubic world around them. Check out our source code
+    <a href="https://github.com/GentenStudios/Phoenix">here</a>.</p>
+    <h2>Quartz Engine</h2>
+    <p>Once Project Phoenix has hit an alpha state, the development of Quarts Engine is planned to begin. Quartz engine
```
Comment (vyomfadia) [inline app/templates/index.html]: Quartz not Quarts

Comment (vyomfadia) [reply app/templates/index.html]: Also capital e in Engine

**Review by vyomfadia — CHANGES_REQUESTED** [review]

File: app/templates/index.html (hunk 1 above)
Comment (apachano) [reply app/templates/index.html]: @beeperdeeper089 Fixed

**Review by vyomfadia — APPROVED** [review]
Make sure @tobyplowy is good with this and feel free to merge

File: app/templates/index.html [hunk 2]
```diff
@@ -2,7 +2,17 @@
 {% block title %}Home{% endblock %}
 {% block body %}
 <div class="jumbotron">
-  <h1>Flask Is Awesome</h1>
-  <p class="lead">And I'm glad to be learning so much about it!</p>
+    <h1>What is Genten Studios?</h1>
+    <p class="lead">Genten Studios is a group of programmers who are passionate about creating games. All of our games
```
Comment (tobyplowy) [inline app/templates/index.html]: "group of programmers"
Hehem... And artists. May wanna say a group of creators instead so it also includes all other categories of contributors.

File: app/templates/index.html [hunk 3]
```diff
@@ -2,7 +2,17 @@
...
+        follow a common pattern that the program itself does not provide any content. Content is defined by modules
```
Comment (tobyplowy) [inline app/templates/index.html:7]: "program itself does not provide any content" may wanna point out the lua interface API.
So maybe like "program itself does not provide any content other then a Lua API.

File: app/templates/index.html [hunk 4]
```diff
@@ -2,7 +2,17 @@
...
+        written in Lua which the game does the heavy lifting turning it into something playable. In the long-term our
+        goal is to create Quartz Engine to be the core of the games we create in this style.</p>
+    <h2>Project Phoenix</h2>
+    <p>Our project coded "Phoenix" is the first game that we are developing in this style. Phoenix is an open world
```
Comment (tobyplowy) [inline app/templates/index.html]: "With this design principle in mind" sounds better then "in this style" (Just an opinion tho.)

File: app/templates/index.html [hunk 5]
```diff
@@ -2,7 +2,17 @@
...
+        sandbox game where the player can manipulate anything in a cubic world around them. Check out our source code
```
Comment (tobyplowy) [inline app/templates/index.html]: "cubic world around them" "In a Voxel based world around them" sounds a bit better and is also more inclusive of the non cubic shapes that are bound to appear and are already present in Phoenix.

File: app/templates/index.html [hunk 6]
```diff
@@ -2,7 +2,17 @@
...
+    <a href="https://github.com/GentenStudios/Phoenix">here</a>.</p>
+    <h2>Quartz Engine</h2>
+    <p>Once Project Phoenix has hit an alpha state, the development of Quartz Engine is planned to begin. Quartz Engine
+        will serve as a common library shared between Phoenix and future games we develop in this same style.</p>
```
Comment (tobyplowy) [inline app/templates/index.html]: "style" Design principle (Check above comments for context)

**Comment by tobyplowy** [comment]
Some comments none of these are essential apart from the first one.

File: app/templates/index.html (hunk 3 above)
Comment (apachano) [reply app/templates/index.html:7]: Is a Lua API content? or is it an API?

Comment (tobyplowy) [reply app/templates/index.html:7]: @apachano It's an API but it is in phoenix/quartz by default. 
Maybe "program itself does not provide any content but it has a Lua feature rich API that can be used to make all kinds of content." ?

File: app/templates/index.html (hunk 2 above)
Comment (apachano) [reply app/templates/index.html]: Fixed

File: app/templates/index.html (hunk 3 above)
Comment (apachano) [reply app/templates/index.html:7]: So the sentence following this one?

File: app/templates/index.html (hunk 4 above)
Comment (apachano) [reply app/templates/index.html]: Sounds good to me

File: app/templates/index.html (hunk 5 above)
Comment (apachano) [reply app/templates/index.html]: Sounds good to me

File: app/templates/index.html (hunk 6 above)
Comment (apachano) [reply app/templates/index.html]: :heavy_check_mark:

File: app/templates/index.html (hunk 3 above)
Comment (tobyplowy) [reply app/templates/index.html:7]: I suppose. You can drop this one.

**Review by tobyplowy — APPROVED** [review]
LGTM

**Pull request merged**